benchmarks/src/print.cpp

```diff
@@ -0,0 +1,32 @@
+﻿#include <benchmark/benchmark.h>
+#include <print>
+
+namespace {
+    using PrintType = void (*)(FILE*, std::string_view, std::format_args);
+
+    template <PrintType PrintFunction>
+    void BM_vprint(benchmark::State& state) {
+        for (auto _ : state) {
+            PrintFunction(stdout, "Hello cool unicode 😊😊😊\n", std::make_format_args());

```

Comment:
Sorry we don't want this 😾

Source files should be plain ASCII

Occurs below

Reply:
Yeah I just saw the bot telling me off, is there a way to test this performance in repo then? Or do we just thoughts and prayers it?

Reply:
I guess I can not do unicode here to be fair, and instead hope somewhere else runs it as a test that unicode console output does still work always

Reply:
`'\x1F60A'` should be fine.

Reply:
Hmm that warns that its too big for a character unfortunately, I'll just remove it and do ! instead, the core here is just if one function is faster than another, not terminal output correctness that should be handled by the actual existing tests

Reply:
Or whatever else way you create any string literals or strings, while keeping the source files and other files ASCII.